### DISCUSSION TITLE
Integration Getting "Stuck"?

### DISCUSSION THREAD

**Discussion by neildhunter** [wills106/homeassistant-solax-modbus#802] [General]
Is anyone else experiencing the integration seemingly getting stuck and not updating entities?
I'm not seeing it lose the entities - they are all still present and showing values, but don't get any updates.

Turning on debug logging I'm seeing the following repeating:
2024-03-11 13:43:15.711 DEBUG (MainThread) [custom_components.solax_modbus] connect modbus
2024-03-11 13:43:20.717 DEBUG (MainThread) [custom_components.solax_modbus] assuming sleep mode - slowing down by factor 10

Reloading the integration itself clears the issue and it starts refreshing the entities regularly again - it's not needing a HA restart etc.

Any thoughts/feedback?!

**Comment by hubertozog** [comment]
Got same issue. Can work few days and stops. In my situation even reboot of HA wont help all the time...

**Comment by adamj89** [comment]
Having the same issue here.

**Comment by ezfrag2021** [comment]
Same here. Having never had to reload the integration previously I now need to do it twice per day to kickstart the entities.
